engine/execution/computation/computer/computer.go

```diff
@@ -399,7 +399,16 @@ func (e *blockComputer) executeTransaction(
 	}
 
 	txView := collectionView.NewChild()
-	err := e.vm.Run(ctx, tx, txView, programs)
+	childCtx := fvm.NewContextFromParent(ctx,
+		fvm.WithLogger(ctx.Logger.With().
+			Str("tx_id", txID.String()).
+			Uint32("tx_index", txIndex).
+			Str("block_id", res.ExecutableBlock.ID().String()).
+			Uint64("height", res.ExecutableBlock.Block.Header.Height).
+			Bool("system_chunk", isSystemChunk).
+			Logger()),
+	)
```

Comment:
Doesn't this duplicate fields?

Reply:
no, In case of a collision, only the last ones are taken.